crates/sierra-casm/src/lib.rs

```diff
@@ -47,3 +45,15 @@ where
     let serialized = serde_json::to_value(value).unwrap();
     serde_json::from_value::<CasmContractClass>(serialized).unwrap()
 }
+
+fn parse_sierra_version(sierra_json: &Value) -> Result<Vec<u8>, String> {
+    let parsed_values: Vec<u8> = sierra_json["sierra_program"]
+        .as_array()
+        .ok_or("Unable to read sierra_program. Make sure it is an array of felts")?
+        .iter()
+        .take(3)
+        .map(|x| u8::from_str_radix(&x.as_str().unwrap()[2..], 16).unwrap_or_default())
```

Comment:
Should we fallback to default and not panic?

Reply:
We can add a comment explaining that I think